src/main/java/org/jabref/model/search/SearchFieldConstants.java

```diff
@@ -9,8 +9,14 @@
 import org.apache.lucene.analysis.en.EnglishAnalyzer;
 
 public enum SearchFieldConstants {
-
-    VERSION("99"),
+    /**
+     * Version number for the search index.
+     * Increment when:
+     * 1. Index changes require reindexing (e.g., new/removed/renamed fields, analyzer changes)
+     * 2. Lucene codec changes (see module-info.java Lucene section)
+     * Incrementing triggers reindexing.
+     */
+    VERSION("1.0"),
```

Comment:
I think, just number is OK.

Alternative: 6.0.0 - to mirror the JabRef main version and the last number is the index version within a role. However since new JabRef versions come out and need no full rebuild, just 1 is good.